README.md

```diff
@@ -165,20 +165,32 @@ A shared resource is helpful for both sides.
 
 ### Legend
 
+Many items covered are marked:
+
 - 🔹 Important and often overlooked tip
 - ❗ Serious warning or danger (where risks or costs are significant)
 - 🔸 A caution, limitation, disadvantage, or quirk
 - 🌪 Controversial topic where informed opinion varies significantly
 - ☝️ Common confusion or misunderstanding, such as confusing terminology
 - ∑ Technical point (arcane or academic and not essential)
 - 📰 New or recent developments
+- 🚧 Expansion or improvement needed (please help!)
+
+You’ll also see links marked:
 
 - 💰 May require payment to read
 - 🔑 May require an account to read
 - 📥 PDF or form or download
 - 🔨 Interactive online tool
 
-- 🚧 Expansion or improvement needed (please help!)
+**A note on reading on Holloway**: Our product is designed to help you find, navigate and
```

Comment:
-would prefer "Reading on Holloway" as the heading (on/on is staccato) 
-serial comma on "navigate"
-change: "find, navigate, and master the material you need." (would consider "grasp" as well)
-"Use the search box." ("and that includes" would modify the "content," just not a clean read)
-So: "Use the search box. It will reveal definitions, section-by-section results, and content contained in the hundreds of resources..."("reveal" or "show you" or "turn out")
-"mini library" = great!!
-serial comma on "section suggestions,"
-do we want to add to the last line: "...useful, or if there's a product feature you'd like to see!" Either way.